docs/conf.py

```diff
@@ -0,0 +1,362 @@
+# -*- coding: utf-8 -*-
+#
+"""Conf file for isbg docs using sphinx."""
+
+# isbg documentation build configuration file, created by
+# sphinx-quickstart on Sat Feb 17 12:46:51 2018.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+import re
+from ast import literal_eval
+import os
+import sys
+import recommonmark
+
+# -- Custom variables -----------------------------------------------------
+cvar_github_base_uri = 'https://github.com/'
+cvar_github_prj = 'isbg'
+cvar_github_usr = 'isbg'
+cvar_github_uri = cvar_github_base_uri + cvar_github_prj + '/' + \
+    cvar_github_usr
+cvar_pypi_uri = 'https://pypi.python.org/pypi/isbg'
+
+master_doc = 'index'  # The master toctree document.
+
+project = u'isbg'
+copyright = u'''License GPLv3: GNU GPL version 3 https://gnu.org/licenses/gpl.html
+
+This is free software: you are free to change and redistribute it. There is
+NO WARRANTY, to the extent permitted by law.'''
+
+author = u'''This software was mainly written Roger Binns <rogerb@rogerbinns.com> and
+maintained by Thomas Lecavelier <thomas@lecavelier.name> since novembre
+2009, and maintained by Carles Muñoz Gorriz <carlesmu@internautas.org>
+since march 2018.'''
```

Comment:
This is likely to change in the future, and it's easy to forget to change it. Couldn't we just refer people to the CONTRIBUTORS file instead?